cmake/modules/FindSUNDIALS.cmake

```diff
@@ -38,11 +38,22 @@
 SET(SUNDIALS_DIR "" CACHE PATH "An optional hint to a SUNDIALS_DIR installation")
 SET_IF_EMPTY(SUNDIALS_DIR "$ENV{SUNDIALS_DIR}")
 
-DEAL_II_FIND_LIBRARY(SUNDIALS_LIB_IDA NAMES sundials_ida
+DEAL_II_FIND_LIBRARY(SUNDIALS_LIB_IDAS NAMES sundials_idas
   HINTS ${SUNDIALS_DIR}
   PATH_SUFFIXES lib${LIB_SUFFIX} lib64 lib
   )
 
+IF(SUNDIALS_LIB_IDAS MATCHES "-NOTFOUND")
+  DEAL_II_FIND_LIBRARY(SUNDIALS_LIB_IDA NAMES sundials_ida
+    HINTS ${SUNDIALS_DIR}
+    PATH_SUFFIXES lib${LIB_SUFFIX} lib64 lib
+    )
+  SET(SUNDIALS_IDA_TYPE "IDA")
+ELSE()
+  ADD_DEFINITIONS(-DDEAL_II_WITH_SUNDIALS_IDAS)
```

Comment:
I think it would be better to add a check in `include/deal.II/base/config.h.in` and explicitly to explicitly set the export in the `config.h` file, instead of using `add_definitions`.

Reply:
I thought about that, but sundials_config.h doesn't export information about what has been built.